docs/RUNNING_ON_A_BUDGET.md

```
@@ -0,0 +1,98 @@
+### OpenCode CLI
+OpenCode is an open-source coding agent that easily routes to custom API providers (like DeepSeek, OpenRouter, Together AI) or local endpoints (Ollama).
+
+To configure OpenCode with a custom provider:
+1. Initialize/open OpenCode in the project directory:
+   ```bash
+   opencode
+   ```
+2. Open its configuration settings (usually located in `.opencode/config.json` or configured via CLI prompts/settings).
+3. Set the `provider` to your chosen endpoint (e.g., OpenRouter or a custom OpenAI-compatible endpoint).
+4. Configure the environment variables for custom endpoints if needed:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)